rollup.build.js

```diff
@@ -0,0 +1,116 @@
+import assets from "postcss-assets";
+import babel from "rollup-plugin-babel";
+import commonjs from "rollup-plugin-commonjs";
+import copy from "rollup-plugin-copy";
+import genHeader from "./dev-lib/header";
+import nodeResolve from "rollup-plugin-node-resolve";
+import packageJSON from "./package.json";
+import postcss from "rollup-plugin-postcss";
+import typescript from "rollup-plugin-typescript2";
+import { readFileSync } from "fs";
+import { terser } from "rollup-plugin-terser";
+
+const babelrc = JSON.parse(readFileSync("./.babelrc"));
+const banner = genHeader("network");
+
+export default [
+  {
+    external: {},
+    globals: {},
+    injectCSS: true,
+    input: "lib/index-standalone.ts",
+    variant: "standalone"
+  },
+  {
+    external: Object.keys(packageJSON.peerDependencies),
+    globals: {
+      keycharm: "keycharm",
+      moment: "moment",
+      "vis-data": "vis",
+      "vis-util": "vis"
+    },
+    injectCSS: false,
+    input: "lib/index-peer.ts",
+    variant: "peer"
+  }
+].flatMap(({ external, globals, injectCSS, input, variant }) => {
```

Comment:
`flatMap` is only supported since node@11 this is a very hard requirement! The last LTS is 10.16. We should find a workaround (also in `examples/network/edgeStyles/arrowAlignment.html:52`).

Reply:
The easiest way would propably be to just use the lodash version.

Reply:
Or simply `[].concat.apply([], arrayOfArray)`.

Reply:
Fixed.